### DISCUSSION TITLE
semver checks not working or a bad config?

### DISCUSSION THREAD

**Discussion by evanchooly** [revapi/revapi#291] [Q&A]
I have revapi configured, i think, to allow for, say, breaking changes on major version bumps but I'm still getting complaints about such breakages.  My config is this:

```json
{
    "extension": "revapi.versions",
    "configuration": {
      "enabled": true,
      "semantic0": true,
      "strictSemver": true,
      "versionIncreaseAllows": {
        "major": {
          "severity": "BREAKING"
        },
        "minor": {
          "severity": "NON_BREAKING"
        },
        "patch": {
          "severity": "EQUIVALENT"
        }
      }
    }
  }
```

What am I missing here?

**Comment by metlos** [comment]
This looks good on its own. Are you able to share the debug output of an occurrence where this happens? I'm especially interested in the "effective configuration" of Revapi and the list of problems that it finds. The effective configuration (along with a ton of other info) is logged on the debug level by the plugin.
